### ISSUE TITLE
French translation dropped ?

### ISSUE THREAD

**Issue by foxmask** [laurent22/joplin#2126]
## Environment

Joplin version: 1.0.311
Platform: Smartphone
OS specifcs: Android

## Steps To Reproduce

1. open Joplin 

then each of the following  reveals a non french translation:

* open a note, then the default title is "add title" and body is "add body"
* when we have a non-encrypted version the orange warning "Press to set a decryption password" is also displayed
* when selecting a note to move it to another folder the popup is "move to notebook" 
* the popup to confirm the deletion says "confirm?" 
* when opening a note, the actions menu on that note contains "attach ...  , set alarm, share, convert to note"
* in the config page the paragraph in english are named "Appearance" , "Plugins"

Describe what you expected to happen: 

everything should be translated ;)

In a previous version, all of that was translated.

**Comment by genneko** [comment]
I just noticed the same issue with Japanese on Joplin Android v1.0.311.
A lot of translations were gone in fdc80f06633202cc86e26d627be4f5ce2a3fbcc4.
I'm not sure it's related to this issue but when I was translating ja_JP.po before, I encountered the similar problem. That was caused by the following bugs in xgettext (on my FreeBSD system) and since then I've been using a manually patched version of gettext when I run Tools/build-translation.js.

- GNU gettext - Bugs: bug #56678, Wrong behaviour of template strings
https://savannah.gnu.org/bugs/?56678
- GNU gettext - Bugs: bug #50920, support Javascript template string in xgettext
https://savannah.gnu.org/bugs/?50920

Hope this helps.
